src/components/tests/content-tab-settings.test.tsx

```diff
@@ -112,6 +111,14 @@ describe( 'ContentTabSettings', () => {
 	beforeEach( () => {
 		jest.clearAllMocks();
 
+		jest.spyOn( global, 'fetch' ).mockResolvedValue( {
+			ok: true,
+			json: jest.fn().mockResolvedValue( {
+				offers: [],
+			} ),
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+		} as any );
```

Comment:
Not sure if it's safe mocking the whole fetch function.
Would be possible to mock just `useGetWordPressVersions`?